File: src/main/kotlin/fr/unilim/saes5/model/reader/TestReader.java
package fr.unilim.saes5.model.reader;

import fr.unilim.saes5.interfaces.IAnalyzable;
import fr.unilim.saes5.model.Word;

import java.util.*;

public class TestReader implements IAnalyzable {

    List<Word> words;

    public TestReader(List<Word> words) {
        this.words = words;
    }
    @Override
    public List<Word> read() {
        return this.words;
    }
}

File: src/main/kotlin/fr/unilim/saes5/model/reader/TestReader.kt
package fr.unilim.saes5.model.reader

import fr.unilim.saes5.interfaces.IAnalyze
import fr.unilim.saes5.model.Word

class TestReader(var words: List<Word>) : IAnalyze {
    override fun read(): List<Word> {
        return words
    }
}
